2.2/.sti/bin/assemble

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+function rake_assets_precompile() {
+  [ -n "$DISABLE_ASSET_COMPILATION" ] && return
+  [ ! -f Gemfile ] && return
+  [ ! -f Rakefile ] && return
+  ! grep " rails " Gemfile.lock >/dev/null && return
+  ! grep " execjs " Gemfile.lock >/dev/null && return
+  ! bundle exec 'rake -T' | grep "assets:precompile" >/dev/null && return
+
+  echo "---> Starting asset compilation."
```

Comment:
Since this is not problem for Ruby 2.0 [1], it should not be blocker for this PR.

[1] https://github.com/openshift/sti-ruby/blob/master/2.0/.sti/bin/assemble#L11